setup.py

```diff
@@ -32,19 +32,25 @@
     'notebook>=5.7.0',
 ]
 
-dev_requires = requires + [
-    'autopep8',
+test_requires = [
     'boto3',
-    'bump2version',
-    'codecov',
-    'flake8',
+    'docker',
     'fs-miniofs',
+    'fs.smbfs @ git+https://github.com/telamonian/fs.smbfs.git@dont_assume_everyone_ace_exists#egg=fs.smbfs',
```

Comment:
Let’s make an issue to either PR this back in or fork to jpmorganchase

Reply:
There already is a PR upstream: althonos /fs.smbfs#11. (Hopefully) the last of three I've submitted to fs.smbfs